content/java/concepts/math-methods/terms/ceil/ceil.md

```diff
@@ -0,0 +1,75 @@
+---
+Title: '.ceil()'
+Description: 'Returns the smallest (closest to negative infinity) double value that is a mathematical integer and greater than or equal to the original value.'
```

Comment:
```suggestion
Description: 'Returns the double value that is a mathematical integer and is greater than or equal to the original value.'
```

Reply:
Can we adjust the intro paragraph to match as well? I removed the part about "closest to negative infinity" because this is actually not the case for negative arguments that get passed into the method.